src/components/AddUserForm.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+type AddUserFormValue = {
+    id: number;
+    lastname: string;
+    name: string;
+    surname: string;
+}
+
+const AddUserForm = ({ onSubmit } : {
+    onSubmit: ( value: AddUserFormValue) => void
+} ) => {
+    const [surnameValue, setSurnameValue] = React.useState('');
+    const [nameValue, setNameValue] = React.useState('');
+    const [lastNameValue, setLastNameValue] = React.useState('');
+
+    return (
+        <div>
+            <form
+                autoComplete="off"
+                onSubmit={event => {
+                    event.preventDefault();
+                    onSubmit({
+                        id: 0,
```

Comment:
you need to use unique ids for each user, or example timestamp or uuid

Reply:
Done